### PR TITLE
Fix  instance_idle issue

### PR DESCRIPTION
**Pull request by athiruma**
## Type of change
Note: Fill **x** in []
- [x] bug
- [ ] enhancement
- [ ] documentation
- [ ] dependencies

## Description

We stopped all running instance

### CODE REVIEW

File: cloud_governance/policy/helpers/aws/aws_policy_operations.py [hunk 1]
```diff
@@ -132,10 +132,8 @@ def update_resource_day_count_tag(self, resource_id: str, cleanup_days: int, tag
             elif self._policy == 'empty_roles':
                 self._iam_client.tag_role(RoleName=resource_id, Tags=tags)
             elif self._policy in ('ip_unattached', 'unused_nat_gateway', 'zombie_snapshots', 'unattached_volume',
-                                  'instance_run'):
+                                  'instance_run', 'instance_idle'):
```
Comment (ebattat) [inline cloud_governance/policy/helpers/aws/aws_policy_operations.py:135]: Why did you delete instance_idle ?

Comment (athiruma) [reply cloud_governance/policy/helpers/aws/aws_policy_operations.py:135]: This is for tagging.

Comment (athiruma) [reply cloud_governance/policy/helpers/aws/aws_policy_operations.py:135]: If the policy is instance_idle, we need to add tags.

Comment (ebattat) [reply cloud_governance/policy/helpers/aws/aws_policy_operations.py:135]: The days count tag ?

Comment (athiruma) [reply cloud_governance/policy/helpers/aws/aws_policy_operations.py:135]: `tags = self._update_tag_value(tags=tags, tag_name='DaysCount', tag_value=str(cleanup_days))`
This code is above, it will take care of updation.
